backend/homework/9/src/main/java/com/example/springhandson3/services/VehicleService.java

```diff
@@ -0,0 +1,29 @@
+package com.example.springhandson3.services;
+
+import com.example.springhandson3.dto.request.AddVehicleRequest;
+import com.example.springhandson3.dto.request.UpdateVehicleRequest;
+import com.example.springhandson3.dto.response.VehicleResponse;
+import com.example.springhandson3.entity.Vehicle;
+
+import java.util.List;
+
+/**
+ * This is the interface for vehicle services
+ */
+public interface VehicleService {
```

Comment:
Interfaces can be store in a sub-package/folder here and are usually prefixed with `I`=> `IVehicleService`